pkgs/tools/misc/aptly/default.nix

```diff
@@ -15,12 +15,17 @@ buildGoPackage rec {
   goPackagePath = "github.com/smira/aptly";
   goDeps = ./deps.nix;
 
+  buildInputs = [ makeWrapper ];
+
   postInstall = ''
     rm $bin/bin/man
+    wrapProgram "$bin/bin/aptly" \
+      --prefix PATH ":" "${gnupg1compat}/bin" \
```

Comment:
Nice! Should I cancel my pull request and make a new one or can I update this one?